corehq/apps/export/views.py

```diff
@@ -1521,6 +1522,58 @@ def get_saved_exports(self):
         return filter(lambda x: x.is_daily_saved_export and x.export_format == "html", combined_exports)
 
 
+@location_safe
+class DataFileDownloadList(BaseProjectDataView):
+    urlname = 'download_data_files'
+    template_name = 'export/download_data_files.html'
+    page_title = ugettext_lazy("Download Data Files")
+
+    def get_context_data(self, **kwargs):
+        context = super(DataFileDownloadList, self).get_context_data(**kwargs)
+        perms = self.request.user.get_all_permissions()
+        context.update({
+            'data_files': DataFile.objects.filter(domain=self.domain).order_by('filename').all(),
+            'can_add': 'export.add_datafile' in perms,
+            'can_delete': 'export.delete_datafile' in perms,
+        })
+        return context
+
+    def post(self, request, *args, **kwargs):
+        # TODO: Use a form for validation
+        # TODO: Limit file size
```

Comment:
im guessing we should look at the hsph domain and see what their requirements are. can't imagine this being larger than 100MB or so, though hsph never ceases to surprise me